### DISCUSSION TITLE
个人空间顶部的背景图屏蔽或者修改

### DISCUSSION THREAD

**Discussion by zjj10330** [the1812/Bilibili-Evolved#2591] [功能建议 / Ideas]
个人空间顶部默认的背景图以白色为主和个人头像右边部分的名称、个人简介都是白色字体，导致很难看清楚，能不能把背景图屏蔽掉或者修改为其他的
![image](https://user-images.githubusercontent.com/21006658/144821140-d0b1675a-4696-41e8-b9d2-60d850172d3e.png)

**Comment by Geeyun-JY3** [comment]
如果只是需要改自己个人空间的头图，你现在就能做到，自定义头图是大会员的权益（换掉这张头图甚至都不需要大会员）：
![image](https://user-images.githubusercontent.com/49396421/144993178-c8cb1d02-65d3-4b3d-83b4-7bbd18d6e69a.png)

**Reply by zjj10330** [reply]
不是修改我自己的，是别人主页的头图，一般很少人会经常点自己的主页的吧
